R/bootstrap.nlsfit.R

```diff
@@ -471,6 +498,24 @@ bootstrap.nlsfit <- function(fn,
 
   errors <- apply(par.boot[rr, , drop=FALSE], 2, error, na.rm = TRUE)
 
+  if(is.null(priors$param)==FALSE && is.null(priors$p)==FALSE && is.null(priors$psamples)==FALSE){
+    lp <- length(priors$p)
+    y <- head(y,-lp)
+    dy <- head(dy,-lp)
+  }
+
+  ## Calculate the energy shifts \Delta E_n in the case of n particle correlators.
```

Comment:
This does not belong into this function. In `bootstrap.nlsfit` we only do a correlated bootstrapped fit to the data using the given model. Here we do not assume any knowledge about the meaning of the model, fit parameters or data.